RUNBOOK.md

```diff
@@ -0,0 +1,81 @@
+## Overview 
+
+The Flow Playground is an web-based interactive IDE for running Cadence code.
+It also provides code for the tutorial projects found here: https://docs.onflow.org/cadence/tutorial/01-first-steps
+
+The overall project consists of the Web app (this) and an API backend: https://github.com/onflow/flow-playground-api
+
+The Playground Web App is implemented in React. The major components are as follows:
+
+- GraphQL / Apollo Client
+  - All HTTP communication with the Playground API is donce via `GraphQL` using the `Apollo` client. 
+  - The GrpahQL schema is defined by the Playground API here: https://github.com/onflow/flow-playground-api/blob/master/schema.graphql
+    - This project uses the Apollo clients localStorage interface as well. 
+    - You can view the _local_ GraphQL schema here: https://github.com/onflow/flow-playground/blob/master/src/api/apollo/local.graphql
+  - CRUD methods (wrapped Apollo client) are implemented here: https://github.com/onflow/flow-playground/blob/master/src/providers/Project/projectMutator.ts 
+  - Typescript typings, and CRUD methods for Apollo are auto-generated using: https://www.graphql-code-generator.com/
+  - After making changes to the `schema.local` you'll need to run `npm run graphql:codegen` to auto-generate new typings and methods for Apollo client.
+
+- Monaco Editor
+  - The editor interface itself is implemented using https://microsoft.github.io/monaco-editor/
+  - The editor component can be found here: https://github.com/onflow/flow-playground/tree/master/src/containers/Editor
+  - The Cadence language definition (for linting and syntax highlighting) for Monaco can be found here: https://github.com/onflow/flow-playground/blob/master/src/util/cadence.ts
+
+- Cadence Language Server
+  - The Cadence Language Server (protocol for Monaco) is implemented in WASM. 
+    - You can read about it here: https://github.com/onflow/cadence/blob/master/languageserver/README.md 
+  - The integration can be found here: 
+      - https://github.com/onflow/flow-playground/blob/master/src/util/language-server.ts
+      - https://github.com/onflow/flow-playground/blob/master/src/util/language-client.ts
+
+## Deployment
+
+### Access
+
+To create a Playground deployment, your user must: 
+- Be added to the Flow/Dapper Labs Teamcity account
+- Have write access to this repository for triggering Teamcity deploys (details below): https://github.com/dapperlabs/flow-playground
+- Be connected to the Office VPN
+
+A request for these should be filed here: https://dapperlabs.happyfox.com/
+
+### Deployment Workflow
```

Comment:
would we want to link to the internal deployment doc? recommend not duplicate contents but aggregate them into one place as one single source of trueth